recipes/stopwordsiso/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "stopwordsiso" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5be2b4f25afc296fca4f626710b98168949057f81a655e2dbdf670ddfce2af50
+
+build:
+  number: 0
```

```suggestion
  number: 0
  noarch: python
```
It seems like this package can be noarch but please double-check this.